primitives/src/network.rs

```diff
@@ -0,0 +1,24 @@
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_runtime::RuntimeDebug;
+
+/// Network type for parallel.
+#[derive(Clone, Copy, RuntimeDebug)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum NetworkType {
+    Parallel,
+    Heiko,
+}
+
+impl NetworkType {
+    /// Return ss58 address prefix from network type.
+    pub fn ss58_addr_format_id(&self) -> u8 {
+        match self {
+            NetworkType::Heiko => HEIKO_PREFIX,
+            NetworkType::Parallel => PARALLEL_PREFIX,
+        }
+    }
+}
+
+pub const HEIKO_PREFIX: u8 = 110;
+pub const PARALLEL_PREFIX: u8 = 172;
```

Comment:
We only have one network called Nodle :)

Reply:
Also our SS58 ID is 37

Reply:
No not relevent to nodle context, clean-up done :+1: